src/auth/index.js

```diff
@@ -13,6 +13,16 @@ const requireAuth = (req, res, next) => {
   return next();
 };
 
+function requireRoles(roles) {
```

Comment:
`require` sounds like **all** roles should be (like `a and b`)
mb `allowRoles` (sounds like `one of (a and b)`),